Community/acl_inventory/__init__.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2018 BlueCat Networks. All rights reserved.
```

Comment:
Please change date to 2019.
There are other instances of 2018 in the other files and once instance of 2017.

Reply:
Dates changed to 2019